user_tools/src/spark_rapids_tools/tools/unsupported_ops_stage_duration.py

```diff
@@ -57,16 +61,20 @@ def __calculate_unsupported_stages_duration(self, unsupported_ops_df: pd.DataFra
         """
         Calculates the percentage of all sql stage durations sum for unsupported operators for each application
         """
-        # Define mask to remove rows invalid entries
+        # Mask defines a set of conditions to filter which unsupported operators have an impact
+        # on the stage duration. Refer to keys unsupportedOperators/mask in qualification-conf.yaml
```

Comment:
`unsupportedOperators/mask` seems confusing. References in yaml files are usually in the `dot` format `<section>.<subsection>.<variable`.  Should we use the complete reference from top level for this?